pylot/planning/frenet_optimal_trajectory/fot_planning_operator.py

```diff
@@ -237,20 +237,29 @@ def _build_obstacle_list(vehicle_transform, prediction_msg):
         # look over all predictions
         for prediction in prediction_msg.predictions:
             # use all prediction times as potential obstacles
+            location = prediction.bounding_box.transform.location
```

Comment:
Can you please ensure it still works if bounding_box is of type BoundingBox2D? It's a bit annoying, but we have to make sure that the planner can use the outputs of our current perception components.

Reply:
Going to revert this change for the C++ port of Frenet. Will add that in the C++ code when it is merged!